### ISSUE TITLE
Use template-literal instead of EJS

### ISSUE THREAD

**Issue by Drulac** [SpencerCDixon/redux-cli#134]
[Template Literal](https://github.com/Drulac/template-literal) is fastest, smallest and simplest template engine, because it use JS's literal template feature.

It's  55 times faster than EJS, and it also use less CPU and RAM ressources, so it may be a good idea to use it instead of EJS 😀

**Comment by anithri** [comment]
I'm not sure the speed of the templating affects the overall runtime very much though, given the amount of file system work being done.  I could be wrong, especially as we add sophistication to blueprints.

We're going to be re-writing the generators anyway, but that needs to be done in tandem with that process.  Or we need to be enable multiple template engines, and a way to define or detect which one to be used.  I kind of like the idea of being able to add other kind of generators (babel, postcss...) to expand on the type's of files we can easily to generate.   

The templating is handled in [src/models/file-info.js](https://github.com/SpencerCDixon/redux-cli/blob/00684d6c16c43bad36d33877967218d54770270d/src/models/file-info.js)

I also suspect that it'd be easier to test and lint template-literals than ejs.  But that's just a gut feeling. 

So I think it would fit, a multi template engine could be coded and added pretty quickly.  That would allow the existing generators to be changed later on.  I'm going to add it to the 2.0 roadmap.

And I'm happy to accept a pull request.

**Comment by Drulac** [comment]
Thanks for enthusiasm 😊 !

It's a good idea to make it become more 

I'm not sur to have skills to make a PR, but I will look 😊

**Comment by SpencerCDixon** [comment]
Yeah I don't think worrying about speed of template rendering for this use case is worth it...  I would be in favor of leaving EJS as it allows you to use expressive logic in your templates.
